lib/networkmonitor.js

```diff
@@ -0,0 +1,58 @@
+'use strict';
+
+
+var bitcore = require('bitcore');
+var Networks = bitcore.Networks;
+var p2p = require('bitcore-p2p');
+var Promise = require('bluebird');
+var $ = bitcore.util.preconditions;
+var _ = bitcore.deps._;
+var EventEmitter = require('events').EventEmitter;
+var util = require('util');
+var EventBus = require('./eventbus');
+var Peer = p2p.Peer;
+var Messages = p2p.Messages;
```

Comment:
Could you sort this list? A good practice is to separate it in 3 groups:

1 - Third party libraries
2 - Our external libraries (bitcore)
3 - Project specific clases

E.g:

```
var util = require('util');
var Promise = require('bluebird');
var EventEmitter = require('events').EventEmitter;

var bitcore = require('bitcore');
var Networks = bitcore.Networks;
var $ = bitcore.util.preconditions;
var _ = bitcore.deps._;
var p2p = require('bitcore-p2p');
var Peer = p2p.Peer;
var Messages = p2p.Messages;

var EventBus = require('./eventbus');
```